### PR TITLE
Appdev 9849 laravel 8 update

### PR DESCRIPTION
**Pull request by sharonluong**
This PR entails the code and composer.json changes needed for Laravel 8. The major changes are:
- composer.json package updates and some new additions
- namespace added to DB seeders, and renaming a directory
- making individual factory files according to new class-based factory structure
- adjusting tests to new factory syntax
- adding trait `HasFactory` to models that have factories
- fixing a routing bug for masters.cuts.create route on Transfers show page

### CODE REVIEW

File: resources/views/transfers/show.blade.php [hunk 1]
```diff
@@ -204,7 +204,7 @@
     <div class="col-xs-12 actions">
       <a class="btn btn-sm btn-secondary" role="button" href="{{ route('transfers.edit', $transfer->id) }}"><i class="fa fa-pencil" aria-hidden="true"></i> Edit</a>
       @if ($transfer->cut === null && $transfer->type === 'Audio') 
-      <a class="btn btn-sm btn-secondary" role="button" href="{{ route('masters.cuts.create', array('masterId' => $transfer->preservationMaster->id, 'transferId' => $transfer->id)) }}"><i class="fa fa-plus" aria-hidden="true"></i> Add Cut</a>
+      <a class="btn btn-sm btn-secondary" role="button" href="{{ route('masters.cuts.create', array('masters' => $transfer->preservationMaster->id, 'transferId' => $transfer->id)) }}"><i class="fa fa-plus" aria-hidden="true"></i> Add Cut</a>
```
Comment (cazzerson) [inline resources/views/transfers/show.blade.php:207]: Why was this change needed?

File: database/factories/AudioVisualItemFactory.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+          'call_number' => 'FS-'.$this->faker->randomNumber(4),
```
Comment (cazzerson) [inline database/factories/AudioVisualItemFactory.php:28]: Why is call_number in this former here but a word in AudioItem?

File: resources/views/transfers/show.blade.php (hunk 1 above)
Comment (sharonluong) [reply resources/views/transfers/show.blade.php:207]: Ah! This was to fix a bug that @ekemeyer noted in the [ticket](https://jira.lib.unc.edu/browse/APPDEV-9849). The logged error was:
`Missing required parameter for [Route: masters.cuts.create]: masters`
I think that it now looks for a more standardized convention when sending in an array of params.

File: database/factories/AudioVisualItemFactory.php (hunk 2 above)
Comment (sharonluong) [reply database/factories/AudioVisualItemFactory.php:28]: I have absolutely no idea. I'll make them both this form.

File: resources/views/transfers/show.blade.php (hunk 1 above)
Comment (cazzerson) [reply resources/views/transfers/show.blade.php:207]: Ah OK, thanks!

**Review by cazzerson — APPROVED** [review]
Looks great!
